app/index.js

```diff
@@ -83,31 +83,31 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'name',
         message: 'What\'s your name:',
         default: this.options.name || this.gitc.user.name,
-        when: this.options.name == null
+        when: !this.options.name
```

Comment:
This will cause a regression by disallowing to pass an empty string as name.

We'll need to keep the `== null` checks. Or we could also do `when: !('name' in this.options)`